lib/internal/promisify.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
```

Comment:
Original code resolves with the array if `fn` callback has more than one non-error argument. 
E.g. for `childProcess.exec` in @chrisdickinson's code promise will be resolved with `[stdout, stderr]`. With your code it will be resolved just with `stdout`.